importer/HkEn.py

```diff
@@ -0,0 +1,80 @@
+import dateparser
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class HkEn(Monument):
+
+    def set_heritage_id(self):
+        wlm_code = self.mapping["table_name"].upper()
+        id_no = "{}-{}".format(wlm_code, self.id)
+        self.add_statement("wlm_id", id_no)
+
+    def set_heritage(self):
+        """
+        Set heritage status.
+
+        If declaration_date present and parseable,
+        add it as start_time qualifier
+        """
+        qualifier = None
+        if self.has_non_empty_attribute("declaration_date"):
+            parsed_d = dateparser.parse(self.declaration_date)
+            # will give None when failure
+            if parsed_d:
+                date_dict = utils.datetime_object_to_dict(parsed_d)
+                qualifier = {"start_time": {"time_value": date_dict}}
+
+        heritage = self.mapping["heritage"]["item"]
+        self.add_statement("heritage_status", heritage, qualifier)
+
+    def set_admin_location(self):
+        """Admin location same for all, no subdivisions."""
+        hkong = "Q8646"
+        self.add_statement("located_adm", hkong)
+
+    def set_location(self):
+        loc_q = None
+        if utils.count_wikilinks(self.location) == 1:
```

Comment:
It seems to be quite common with multiple links where one of them is the link to a district ([query for all 18](https://query.wikidata.org/#%23Cats%0ASELECT%20%3Fitem%20%3FitemLabel%20%0AWHERE%20%0A{%0A%20%3Fitem%20wdt%3AP31%20wd%3AQ50256.%0A%20%3Fitem%20wdt%3AP131*%20wd%3AQ8646.%0A%20SERVICE%20wikibase%3Alabel%20{%20bd%3AserviceParam%20wikibase%3Alanguage%20%22en%22.%20}%0A})).

I would suggest checking if any of the links correspond to one of these. If so add that as admin_location and remove it from the list of detected links. if only one remains then use that for location.

If this is called before set_admin_location then that can quickly detect if a value has been added and only go through with the defaults otherwise.